src/loaders/OMETiffLoader.js

```diff
@@ -354,13 +363,14 @@ export default class OMETiffLoader {
   }
 
   /**
-   * Trunactes tiles on the edge to match the height/width reported by the image.
+   * For a given resolution level, z, the expected tile size on the boundary
+   * of the image should be exactly enough to fit the image bounds at the resolution level.
+   * This function returns that size or the parametrized tileSize from TIFF file.
    * @param {tileData: TypedArray[]} data The array to be filled in.
    * @param {Object} tile { x, y, z }
    * @returns {TypedArray} TypedArray
    */
-  _truncateTiles(data, tile) {
-    const { x, y, z } = tile;
+  _getExpectedTileSizeForTiff({ x, y, z }) {
```

Comment:
nit, maybe `_getTileExtent`? It's on the tiff loader so I don't know what `ForTiff` adds for clarity.

Reply:
I figured it meant tiff as in the tiff file, as opposed to image which is an entity within a tiff.  Nonetheless, this is much cleaner